src/bindgen/ir/item.rs

```diff
@@ -75,18 +75,20 @@ pub enum ItemValue<T: Item> {
 
 #[derive(Debug, Clone)]
 pub struct ItemMap<T: Item> {
-    data: BTreeMap<Path, ItemValue<T>>,
+    data: IndexMap<Path, ItemValue<T>>,
```

Comment:
Why is `IndexMap` needed? Not necessarily opposed to this, but unless I'm missing something this change is kind of unrelated and should probably be done in a different PR.